pgcli/packages/pgspecial/main.py

```diff
@@ -18,6 +18,32 @@
 class CommandNotFound(Exception):
     pass
 
+
+@export
+class PGSpecial(object):
+    def __init__(self):
+        self.timing_enabled = True
+
+        self.commands = COMMANDS.copy()
```

Comment:
Once we take the route of using a class, I'd prefer not to maintain a global COMMANDS var. It'll be better to just encapsulate it as a member var in the class.